src/lua/compat.c

```diff
@@ -6,6 +6,24 @@
 
 #include <lua/compat.h>
 
+static const char escaped_slash[] = "\\/";
+
+int
+msgpuck_esc_slash_toggle(struct lua_State *L)
+{
+	assert(lua_isboolean(L, -1) && "boolean argument expected");
+	bool val = lua_toboolean(L, -1);
+	lua_pop(L, 1);
+	lua_pushnumber(L, 0);
```

Comment:
Why we need to push zero?

Reply:
Answered in https://github.com/tarantool/tarantool/pull/6910#discussion_r981081727.

Reply:
Still, segfault.

Reply:
U were right, removed pushnull and changed `return 1` to `return 0`

Reply:
It is fixed now.